config/ant-phase-verify.xml

```diff
@@ -38,7 +37,6 @@
       <formatter type="plain"/>
       <formatter type="xml" toFile="${mvn.project.build.directory}/cs_errors.xml"/>
       <formatter type="sarif" toFile="${mvn.project.build.directory}/cs_errors.sarif"/>
-      <classpath path="${mvn.runtime_classpath}"/>
```

Comment:
Assume this is just left over when we had checks against the classpath.

Reply:
I thought it is required, as we load all check by classpath not a jar

Reply:
Please note this is a custom property sent to our Checkstyle Ant. It is not related to Java's classpath loader.
https://github.com/checkstyle/checkstyle/blob/527578f9a89abd7136552763bf942f9cb334c203/src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java#L92

Java's classpath loader is at https://github.com/checkstyle/checkstyle/blob/af99c7d552ee8470cd07590211e6cdf1a8881050/config/ant-phase-verify.xml#L12 .

If you examine the code, you will see this property is only used to display as an error if an exception occurs.
https://github.com/checkstyle/checkstyle/blob/527578f9a89abd7136552763bf942f9cb334c203/src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java#L430-L431

Technically this whole property can be marked as deprecated.

I haven't searched the history, but this seems like its just for check's to access the real classpath.

Reply:
https://github.com/checkstyle/checkstyle/blob/b8ca6a585b824e91b3b8c72dd5cc53c0eb0ab0f1/src/main/java/com/puppycrawl/tools/checkstyle/CheckStyleTask.java#L362-L365
Original usage of the classpath property that is now removed.

Reply:
Code was removed at 
https://github.com/checkstyle/checkstyle/commit/145160f5e21b80c27dc93a1904fe33b9afd4f212#diff-56ee8828e56742120dd2db0061ea82707b7b27c5c0f9e9963a8cefe3aa2dbc3aL405
as part of https://github.com/checkstyle/checkstyle/issues/3773